scripts/repl.js

```diff
@@ -284,7 +371,34 @@
   /*
    * Initialize the REPL
    */
-  var repl = new REPL();
+  var initialQuery = UriUtils.parseQuery();
+  if (initialQuery && initialQuery.short) {
```

Comment:
It doesn't require the url to be shortened, it's just handling the case where it is.  Otherwise it falls through to the existing behavior.